providers/baseprov.c

```diff
@@ -80,15 +80,11 @@ static const OSSL_ALGORITHM base_encoder[] = {
 #undef ENCODER
 
 static const OSSL_ALGORITHM base_decoder[] = {
-#define DECODER(name, _fips, _input, func_table)                            \
-    { name,                                                                 \
-      "provider=base,fips=" _fips ",input=" _input,                         \
-      (func_table) }
-
+#define DECODER_PROVIDER "base"
 #include "decoders.inc"
     { NULL, NULL, NULL }
 };
-#undef DECODER
+#undef DECODER_PROVIDER
```

Comment:
I would have put the #undef below the #include if you are goin to embed the define inside the block.

Reply:
Okie

Reply:
I suppose that you would like the same thing to happen in #13167